includes/islandora_scan_batch.inc

```diff
@@ -528,4 +542,12 @@ EOXML;
   protected function addContentModelRelationships() {
     $this->models = $this->preprocessorParameters['content_models'];
   }
+
+  /**
+   * Accessor for content models.
+   */
+  public function getContentModels() {
```

Comment:
Is this really necessary? `$this->models` should already be available anywhere it needs to be gotten.